src/server/network/protocol/protocolgame.cpp

```diff
@@ -6297,6 +6402,16 @@ void ProtocolGame::AddPlayerSkills(NetworkMessage &msg)
 	// used for imbuement (Feather)
 	msg.add<uint32_t>(player->getCapacity()); // total capacity
 	msg.add<uint32_t>(player->getBaseCapacity()); // base total capacity
+
+	// 12.81 Protocol
+	msg.add<uint16_t>(0);
+	msg.add<uint16_t>(0);
+
+	msg.add<uint16_t>(0);
+	msg.add<uint16_t>(0);
+
+	msg.add<uint16_t>(0);
+	msg.add<uint16_t>(0);
```

Comment:
move `cap` and `bonus cap` to the end of packet or the client will read them as base/bonus `onslaught`